code/modules/economy/account.dm

```diff
@@ -54,12 +54,16 @@
 	bank_card_talk("ERROR: Payday aborted, unable to contact departmental account.")
 	return FALSE
 
-/datum/bank_account/proc/bank_card_talk(fuck)
-	if(!fuck || !bank_cards.len)
+/datum/bank_account/proc/bank_card_talk(message)
+	if(!message || !bank_cards.len)
 		return
 	for(var/obj/A in bank_cards)
-		for(var/mob/M in hearers(1,get_turf(A)))
-			to_chat(M, fuck)
+		if(ismob(A.loc))
```

Comment:
This won't get the loc if it's inside a bag or PDA or box inside a bag, or PDA inside a box inside a bag, I think. Which was stopping me from just doing this before.

Reply:
Oh, i see now. I think someone made a proc for this, i'll see if i can find it.

Reply:
Also, do me a solid and swap it to this:
``		playsound(A, 'sound/machines/twobeep.ogg', 50, 1)
		A.audible_message("[icon2html(A, hearers(A))] *[fuck]*", null, 1)``

Reply:
if not the audible_message part, keep the sound and image, visual/audio feedback is important

Reply:
alright